src/main/java/com/paneedah/mwc/tileentities/TileEntities.java

```diff
@@ -14,2006 +14,2002 @@
 public class TileEntities {
 
     public static void init(CommonProxy commonProxy) {
-    	new TurretBaseFactory().createTileEntity(ModernWarfareMod.MOD_CONTEXT);
```

Comment:
Not related to props and anyway should not be removed as I said in https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/54